common/thorhelper/thorcommon.hpp

```diff
@@ -89,7 +89,9 @@ enum RowReaderWriterFlags
     rw_compressblkcrc = 0x10, // block compression, this sets/checks crc's at block level
     rw_fastlz         = 0x20, // if rw_compress
     rw_autoflush      = 0x40,
-    rw_buffered       = 0x80
+    rw_buffered       = 0x80,
+    rw_lz4            = 0x100, // if rw_compress
+    rw_den            = 0x200  // if rw_compress
```

Comment:
To avoid having different flags, e.g. rw_fastlz , COMPRESS_METHOD_FASTLZ  and rw_lz4 + COMPRESS_METHOD_LZ4 etc.

It's probably worth sticking to just the COMPRESS_METHOD_XXX flavours, getting rid of rw_fastlz,rw_lz4,rw_den and using a macro to bit shift them into the flags here.
e.g. rwCompFlag(COMPRESS_METHOD_LZ4)

Related to the mask point.